### DISCUSSION TITLE
Should `stabilityDays` work with Docker images?

### DISCUSSION THREAD

**Discussion by benedfit** [renovatebot/renovate#22669] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Should Docker image updates adhere to the `stabilityDays` config rule? Do the Docker images need some extra meta information in order for this to work?

We've noticed that one docker image that's grouped together with some NPM updates always gets created instantly rather than waiting for the 3 day stability window. If we don't merge it, it eventually does get grouped with the other updates, but it usually auto-merges before then

### What you tried so far.

Tried specifically adding `stabilityDays` to the individual dependency, but to no avail

### Relevant debug logs

<details><summary>Logs</summary>

| Package | Type | Update | Change |
|---|---|---|---|
| mcr.microsoft.com/playwright | docker | minor | `v1.34.3-focal` -> `v1.35.0-focal` |

```
DEBUG: CircleCI docker image
{
  "depName": "mcr.microsoft.com/playwright"
  "currentValue": "v1.34.3-focal"
}
```

</details>

**Comment by rarkins** [comment, marked as answer]
It's expected behavior that it doesn't work. Depends on https://github.com/renovatebot/renovate/issues/5773
